docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -115,82 +119,133 @@ POST /transactions/
         "version": 1
       }
 
+   :resheader Content-Type: ``application/json``
+
+   :statuscode 200: A transaction with that ID was found.
+   :statuscode 404: A transaction with that ID was not found.
+
+.. http:get:: /transactions
+
+   The current ``/transactions`` endpoint returns a ``404 Not Found`` HTTP
+   status code. Eventually, this endpoint might get implemented, its usefulness
+   would be questionable however, as the transactions listed would change by
+   the second.
+
+   There are however requests that might come of use, given the endpoint is
+   queried correctly. Some of them include retrieving a list of transactions
+   that include:
+
+   * `Unfulfilled conditions <#get--transactions?fields=id,conditions&fulfilled=false&owner_afters=owners_after>`_
+   * `A specific asset <#get--transactions?fields=id,asset,operation&operation=CREATE|TRANSFER&asset_id=asset_id>`_
+   * `Specific metadata <#get--transactions?fields=id,metadata&metadata_id=metadata_id>`_
+
+   In this section, we've listed those particular requests, as they will likely
+   to be very handy when implementing your application on top of BigchainDB.
+   A generalization of those parameters can follows:
+
+   :query fields: A comma separated string to expand properties on the transaction object to be returned.
+   :type fields: string
+
+   :query fulfilled: A flag to indicate if transaction's with fulfilled conditions should be returned.
+   :type fulfilled: boolean
+
+   :query owners_after: Public keys able to validly spend an output of a transaction, assuming the user also has the corresponding private key.
+   :type owners_after: base58 encoded string
```

Comment:
`get_owned_ids` just takes a single owner, but for the API we could allow multiple keys.
Right now, we don't have a use case to support multiple keys as far as I'm aware of.

My suggestion:

Keep it a single key for now. Allow multiple keys as soon as someone asks for it, as this functionality can be added without breaking the input and output of the endpoint.
OK?